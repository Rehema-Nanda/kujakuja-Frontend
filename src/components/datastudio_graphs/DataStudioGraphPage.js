import React from 'react';
import AppConfig from "../../AppConfig";

const DataStudioGraphPage = () => {
    return (
        AppConfig.DATA_STUDIO_GRAPH_URL && (
            <div style={{ textAlign: 'center', margin: '10px 0' }}>
                <iframe
                    title="graph"
                    width="1000"
                    height="800"
                    src={AppConfig.DATA_STUDIO_GRAPH_URL}
                    frameBorder="0"
                    style={{ border: 0, margin: 'auto' }}
                    allowFullScreen
                />
            </div>
        )
    );
};

export default DataStudioGraphPage;
