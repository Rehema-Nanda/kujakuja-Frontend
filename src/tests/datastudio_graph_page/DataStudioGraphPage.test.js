import React from "react";
import { mount } from "enzyme";
import { BrowserRouter } from "react-router-dom";
import DataStudioGraphPage from "../../components/datastudio_graphs/DataStudioGraphPage";

describe("DataStudioGraphPage component", () => {
    const wrapper = mount(<BrowserRouter><DataStudioGraphPage /></BrowserRouter>);

    it("render correctly", () => {
        expect(wrapper.debug()).toMatchSnapshot();
    });
});
