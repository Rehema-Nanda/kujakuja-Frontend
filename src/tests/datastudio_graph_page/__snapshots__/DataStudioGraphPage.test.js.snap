// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataStudioGraphPage component render correctly 1`] = `
"<BrowserRouter>
  <Router history={{...}}>
    <DataStudioGraphPage>
      <div style={{...}}>
        <iframe title=\\"graph\\" width=\\"1000\\" height=\\"800\\" src=\\"https://kujakuja.com\\" frameBorder=\\"0\\" style={{...}} allowFullScreen={true} />
      </div>
    </DataStudioGraphPage>
  </Router>
</BrowserRouter>"
`;
